### DISCUSSION TITLE
Anyone else having this problem with Yahoo! Sports?

### DISCUSSION THREAD

**Discussion by tsuliwaensis** [darkreader/darkreader#8592] [Q&A]
Using Dark Reader at https://sports.yahoo.com, when hovering over a headline, the area under the cursor turns black. You can see this with the video I took [https://imgur.com/DpD3pWS](https://imgur.com/DpD3pWS). Does this happen to anyone else?

**Comment by Gusted** [comment]
Don't use discussions for bug reports.
